src/allotropy/parsers/moldev_softmax_pro/softmax_pro_parser.py

```diff
@@ -264,3 +287,63 @@ def _iter_absorbance_measurement_document(
                     ]
                 ),
             )
+
+    def _iter_calculated_data_documents(
+        self, data: Data
+    ) -> Iterator[CalculatedDataDocumentItem]:
+        for plate_block in data.block_list.plate_blocks.values():
+            for reduced_data_element in plate_block.iter_reduced_data():
+                yield CalculatedDataDocumentItem(
+                    calculated_data_identifier=str(uuid.uuid4()),
+                    calculated_data_name="Reduced",
```

Comment:
Will we be using "Reduced" in other parsers? If so, should we make a constant for this?

Reply:
Reduced is specific to this parser. I put a constant for this value.